README.md

```diff
@@ -125,173 +125,6 @@ Find the **Protocol Buffers Descriptions** in the [`./pb` directory](./pb).
 * **Synthetic Load Generation:** The application demo comes with dedicated load generation service that creates realistic usage patterns on Hipster Shop website using
   [Locust](https://locust.io/) load generator.
 
-## For Developers
```

Comment:
Add a note - if developers want to contribute to Sandbox, they should refer to CONTRIBUTING.md (also link to it)